src-plugins/vtkDataMesh/writers/vtkDataMeshWriter.cpp

```diff
@@ -53,8 +59,6 @@ bool vtkDataMeshWriter::write(const QString& path)
   if (!this->data())
     return false;
 
-  qDebug() << "Can write with: " << this->identifier();
-
   medAbstractData * medData = dynamic_cast<medAbstractData*>(this->data());
```

Comment:
It's a bit complicated what you asked @LoicCadour. If i move the metaData functions into vtkMetaSurfaceMesh i do not have access anymore to `this->data()` from where i've got the metadata information to write. Everything looks fine but i just need to access to `this->data()`. 

So, now, i've got the "read" functions in vtkMetaSurfaceMesh and the "write" functions in vtkDataMeshWriter, which is kind of ugly i think. I would prefer something symmetrical: either everything in vtkMetaSurfaceMesh or vtkDataMeshWriter.

What do you think? I would let like that here ^^